src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -70,7 +70,7 @@ public DatabaseRecord(string databaseName)
 
         public ClientConfiguration Client;
 
-        public long TruncatedClusterTransactionIndex;
+        public long TruncatedClusterTransactionCommandsCount;
```

Comment:
This is serialized, right? If so, it is fine to make a change here, since 4.1 isn't released yet, but need to make sure to remember this in the future.

Reply:
Note that our production system already has a 4.1 instance running tryouts, so need to check that this won't impact it there.

Reply:
Good point. Don't think that this property has propagated to the DatabaseRecord into production.
(haven't saw it on the DatabaseRecord)